plugins/admin/pkg/command/domain/set_test.go

```diff
@@ -0,0 +1,227 @@
+				Name:      "config-domain",
+				Namespace: "knative-serving",
+			},
+			Data: map[string]string{
+				"dummy.domain": "",
+			},
+		}
+		client := k8sfake.NewSimpleClientset(cm)
+		p := pkg.AdminParams{
+			ClientSet: client,
+		}
+		cmd := NewDomainSetCommand(&p)
+
+		_, err := testutil.ExecuteCommand(cmd, "--custom-domain", "dummy.domain")
+		assert.NilError(t, err)
+
+		updated, err := client.CoreV1().ConfigMaps("knative-serving").Get("config-domain", metav1.GetOptions{})
+		assert.NilError(t, err)
+		assert.Check(t, equality.Semantic.DeepEqual(updated, cm), "configmap should not changed")
+
+	})
+
+	t.Run("adding domain config without selector with existing domain configuration", func(t *testing.T) {
+		cm := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "config-domain",
+				Namespace: "knative-serving",
+			},
+			Data: map[string]string{
+				"foo.bar": "",
+			},
+		}
+		client := k8sfake.NewSimpleClientset(cm)
+		p := pkg.AdminParams{
+			ClientSet: client,
+		}
+		cmd := NewDomainSetCommand(&p)
+		o, err := testutil.ExecuteCommand(cmd, "--custom-domain", "dummy.domain")
+		assert.NilError(t, err)
+		assert.Check(t, strings.Contains(o, "Updated knative route domain to"), "expected update information in standard output")
+
+		cm, err = client.CoreV1().ConfigMaps("knative-serving").Get("config-domain", metav1.GetOptions{})
+		assert.NilError(t, err)
+		assert.Check(t, len(cm.Data) == 1, "expected configmap lengh to be 1, actual %d", len(cm.Data))
+
+		v, ok := cm.Data["dummy.domain"]
+		assert.Check(t, ok, "domain key %q should exists", "dummy.domain")
+		assert.Equal(t, v, "", "value of key domain should beempty")
+	})
+
+	t.Run("adding domain config with selector", func(t *testing.T) {
+		cm := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "config-domain",
+				Namespace: "knative-serving",
+			},
+			Data: map[string]string{
+				"foo.bar": "",
+			},
+		}
+		client := k8sfake.NewSimpleClientset(cm)
+		p := pkg.AdminParams{
+			ClientSet: client,
+		}
+		cmd := NewDomainSetCommand(&p)
+
+		o, err := testutil.ExecuteCommand(cmd, "--custom-domain", "dummy.domain", "--selector", "app=dummy")
+		assert.NilError(t, err)
+		assert.Check(t, strings.Contains(o, "Updated knative route domain to"), "invalid output %q", o)
+
+		cm, err = client.CoreV1().ConfigMaps("knative-serving").Get("config-domain", metav1.GetOptions{})
+		assert.NilError(t, err)
+		assert.Check(t, len(cm.Data) == 2, "expected configmap lengh to be 2, actual %d", len(cm.Data))
+
+		v, ok := cm.Data["dummy.domain"]
+		assert.Check(t, ok, "domain key %q should exists", "dummy.domain")
+
+		var s domainSelector
+		err = yaml.Unmarshal([]byte(v), &s)
+		assert.NilError(t, err)
+		assert.Check(t, len(s.Selector) == 1, "selector should only contain one key-value pair, got %+v", s.Selector)
+
+		v, ok = s.Selector["app"]
+		assert.Check(t, ok, "key %q should exist", "app")
+		assert.Equal(t, v, "dummy")
```

Comment:
@lanceliuu the assert works here, but I think we need to follow the function definition here, would you please help to switch the expected and actual value in this PR. Thanks!
```
func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool
```

Reply:
Sure. I have changed all the assert.Equal calls to follow the expected signature.